src/vbe_info.rs

```diff
@@ -0,0 +1,131 @@
+use core::fmt;
+
+// Use this for implementing Debug on fields which are too large, or should not be printed.
+// Although I haven't found anything explicitly saying that this will _definitely_ have the same
+// representation as T, in my tests it always has. I don't know quite how alignment works either
+// for this type, the rust docs aren't totally clear, and #[repr(C, packed)] is not valid.
+#[derive(Copy, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
+struct NoDebug<T>(T);
+
+impl<T> fmt::Debug for NoDebug<T> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "...") // Write something, otherwise this looks funky in structs.
+    }
+}
+
+#[derive(Debug, Copy, Clone)]
+#[repr(C, packed)]
+pub struct VBEInfoTag {
+    typ: NoDebug<u32>,
+    length: NoDebug<u32>,
```

Comment:
Initially I did that because they're not really relevant to the contents of the struct, but since we read it directly out of memory, that would be good for debugging, so i removed the NoDebug.